UPDATING.md

```diff
@@ -62,6 +62,41 @@ https://developers.google.com/style/inclusive-documentation
 
 -->
 
+### DAG.create_dagrun accepts run_type and does not require run_id
+This change is caused by adding `run_type` column to `DagRun`.
+
+Previous signature:
+```python
+def create_dagrun(self,
+                  run_id,
+                  state,
+                  execution_date=None,
+                  start_date=None,
+                  external_trigger=False,
+                  conf=None,
+                  session=None):
+```
+current:
+```python
+def create_dagrun(self,
+                  state,
+                  execution_date=None,
+                  run_id=None,
+                  start_date=None,
+                  external_trigger=False,
+                  conf=None,
+                  run_type=None,
```

Comment:
Run_type should probably be required, no?

Reply:
`run_type` can be derived from `run_id` too so I think the default of None is correct. Just needs to error when both of them are `None`

Reply:
And that's checked:
```python
        elif not run_id:
             raise AirflowException(
                  "Creating DagRun needs either `run_id` or `run_type` and `execution_date`"
             )
```